src/sentry/models/integration.py

```diff
@@ -51,7 +51,9 @@ class RepositoryProjectPathConfig(DefaultFieldsModel):
 
     repository = FlexibleForeignKey("sentry.Repository")
     project = FlexibleForeignKey("sentry.Project", db_constraint=False)
-    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
+    organization_integration = FlexibleForeignKey(
+        "sentry.OrganizationIntegration", null=True, on_delete=models.SET_NULL
```

Comment:
Just checking that `SET_NULL` is really what you want here? So you want a `RepositoryProjectPathConfig` row to be orphaned from that relationship when a related `OrganizationIntegration` is deleted, is that correct?

Reply:
@wedamija ahh yes that's what I wanted, but I just spoke with @MeredithAnya about it and agreed that its better to delete the `RepositoryProjectPathConfig` row. I'll make an update for it.